packages/insomnia-app/app/plugins/install.ts

```diff
@@ -56,10 +57,15 @@ export default async function(lookupName: string) {
       mkdirp.sync(pluginDir);
 
       // Download the module
-      const request = electron.remote.net.request(info.dist.tarball);
-      request.on('error', err => {
-        reject(new Error(`Failed to make plugin request ${info?.dist.tarball}: ${err.message}`));
-      });
+      let request;
+      try {
+        request = await axiosRequest({ url: info.dist.tarball });
+      } catch (err) {
+        request.error = err;
```

Comment:
Seems like we can remove the `.error` property from the request and directly throw here:

```suggestion
        reject(new Error(`Failed to make plugin request ${info?.dist.tarball}: ${err.message}`));
```

Reply:
thanks